File: cop3223H/Code/roadtrip.py
#Federico Baron
#8/21/18
#Roadtrip calculates the amount of money to be spent on the trip
#gasPrice float is the price of gasoline in dollars
#gallonsInCar is a float and is the amount of gallons left in the car
#mpg float is the miles per gallon of the car
#tripLength is the length of the trip in miles
#total float is the money that is needed to complete the trip
gasPrice=float(input("What is the price of gasoline in dollars?"))
gallonsInCar=float(input("How many gallons of gas are in the car?"))
mpg=float(input("how many miles per gallon does your car get?"))
tripLength=float(input("how long is your road trip in miles?"))
total=((tripLength/mpg)-gallonsInCar)*gasPrice
print("you will need to spend $"+"{0:.2f}".format(total)+" on extra gas to complete the trip")

File: cop3223H/Code/roadtrip.c
//Federico Baron
//8/22/18
//calculates amount of money required for the roadtrip
#include <stdio.h>
int main(void) {
  //gasPrice is the price of gas per gallon
  //gasInCar is the amount of fuel left in the car in gallons
  //mpg is the miles per gallon the car does
  //tripLen is how long the trip is in miles
  //total is the calculation of how much money will need to be spent
  double gasPrice, gasInCar, mpg, tripLen;
  printf("What is the price of gasoline?\n");
  scanf("%lf",&gasPrice);
  printf("How many gallons of gas are in the car?");
  scanf("%lf",&gasInCar);
  printf("How many miles per gallon does your car get?");
  scanf("%lf",&mpg);
  printf("How long is your road trip in miles?");
  scanf("%lf",&tripLen);
  //calculates total to be spent
  double total = (tripLen/mpg-gasInCar)*gasPrice;
  printf("You need to spend $ %.2lf on extra gas to complete the trip.", total);
  return 0;
}